bin/minifollowups/pycbc_plot_trigger_timeseries

```diff
@@ -39,7 +40,8 @@ parser.add_argument('--times', nargs='+', type=float,
 parser.add_argument('--special-trigger-ids', nargs='+', type=int,
     action=MultiDetOptionAction, metavar="IFO:GPS_TIME",
     help="The set of special trigger ids to plot a star at")
-parser.add_argument('--plot-type', choices=['snr', 'newsnr'], default='snr',
+parser.add_argument('--plot-type',
+    choices=ranking.sngls_ranking_function_dict, default='snr',
```

Comment:
I suggest updating the help string as well here, maybe "Which single-detector ranking statistic to plot".